meson.build

```diff
@@ -3,10 +3,15 @@
 project(
   'pymmcore-nano',
   'cpp',
-version : '11.5.0.73.0',
+version : '11.5.1.73.0',
   meson_version : '>= 1.2.0',
   default_options : ['cpp_std=c++17']
 )
+
+if host_machine.system() == 'darwin'
+  add_project_arguments('-U_LIBCPP_ENABLE_ASSERTIONS', language: 'cpp')
```

Comment:
I'm wondering about what caused this to be needed because it's a bad sign. Is there a test that fails that I can use to reproduce?

Reply:
it's not a test fail, just a compile time warning.  You should just be able to clone this repo on your mac, remove this line, run `just clean` and `just install` and you should should see it

Reply:
possibly related? https://github.com/mesonbuild/meson/issues/14440

Reply:
Oooh, I didn't know they had changed the name of the macro. Got it, thanks.